src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
Because it's TitleCase. For someone familiar with javascript that usually means it has to be instantiated.